recipes/hitac/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "1.0" %}
+
+package:
+  name: hitac
+  version: {{ version }}
+
+build:
+  number: 0
+  noarch: python
+source:
+  url: https://gitlab.com/dacs-hpi/hitac/uploads/ea79d6d24da7b72038199f2cccebf724/hitac
+  md5: 94cc863fa214582c8ac50fdcb9347412
```

Comment:
Please use a SHA256 checksum instead.

Reply:
Done! Now it is sha256: d7152a37cae58ef1932a586fd10bf37349a3e5969eaed23f2ac9e276565bd4b4